### ISSUE TITLE
DropdownMenu arrow navigation doesn't work with up/left arrow keys

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1874]
The DropdownMenu component, currently used in the Table Block, implements arrow navigation, which is pretty good for accessibility. Pressing the DOWN or RIGHT arrow keys, the next item in the menu gets focused:

![screen shot 2017-07-12 at 18 04 32](https://user-images.githubusercontent.com/1682452/28136005-88457d6c-6748-11e7-9726-bbd666f5ac41.png)

However, when pressing the UP or LEFT arrow keys, focus is lost.
